frontend/src/components/Groups/CreateGroup.js

```diff
@@ -41,6 +42,7 @@ const ButtonGroup = styled(Box)({
 })
 
 const CreateGroupPage = (props) => {
+    const user = useAuth().currentUser
```

Comment:
I think the recommended way is to use destructuring assignment: const { currentUser } = useAuth(). Unless for some reason you really like the variable name user over currentUser.